scripts/overton/EDA_script.py

```diff
@@ -0,0 +1,176 @@
+import sys  
+
+import pandas as pd
+from collections import Counter
+import numpy as np
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+import spacy
+import seaborn as sns
+import typer
+
+#EDA_FOR_NLP_PATH = 'C:/Users/Hubert/Dropbox/DarlingProject/eda_for_nlp_package'
+#FINAL_TABLE_PATH = 'C:/Users/Hubert/Documents/DarlingProject/Overton/EDA/final_table.parquet'
+#SPACY_TABLE_PATH = 'C:/Users/Hubert/Documents/DarlingProject/Overton/EDA/spacy_table_new.parquet'
+
+def main(eda_for_nlp_path : str, final_table_path : str, spacy_table_path : str):

```

Comment:
i tu też prośba o więcej printów z tym co się aktualnie liczy, ułatwia to zlokalizowanie długo liczących się kawałków np przy optymalizacji